### PR TITLE
All tests

### PR DESCRIPTION
**Pull request by aljones15**
@msporny 

ok this is all of the tests together in one branch with the new report in there too.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,28 +1 @@
...
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+All documents in this Repository are licensed by contributors under the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
```
Comment (mattcollier) [inline LICENSE]: I think this becomes LICENSE.md since it includes markdown.

Comment (aljones15) [reply LICENSE]: Addressed in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/91c5175645c2a65ed730f849a605839af7d6e590

File: implementations/generate.js [hunk 2]
```diff
@@ -6,108 +6,75 @@
...
+//used to get the title of all tests for the spec
+function makeMaster(masterName) {
+  const context = {
+    implementations: [],
+    tests: {}
+  };
+  const masterFile = `${masterName}-report.json`;
+  const masterPath = files.find(f => f === masterFile);
+  if(!masterPath) {
+    throw new Error(`Unable to find json report at ${masterPath}
```
Comment (mattcollier) [inline implementations/generate.js]: The error message using backticks like this is going to have extra whitespace.  Please change to use + on line continuation.

File: implementations/implementation.hbs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      </p>
+
+      <section class="informative">
+        <h2>Testing Methodology</h2>
+
+        <p>
+The testing framework for HTTP Signatures executes the
+following process for every conformance statement in the 
+<a href="https://tools.ietf.org/html/draft-cavage-http-signatures">
+Http Signatures specification</a> :
```
Comment (mattcollier) [inline implementations/implementation.hbs]: Http -> HTTP

File: test/latest/10-canonicalize.js [hunk 4]
```diff
@@ -5,14 +5,19 @@ const {expect} = require('chai');
...
-  before(function() {
+  beforeEach(function() {
     generatorOptions = {
       generator: config.generator,
       command: 'c14n',
       args: {},
       date: new Date().toGMTString(),
     };
   });
+  describe('Optional', function() {
```
Comment (mattcollier) [inline test/latest/10-canonicalize.js]: Should this be skipped or a todo?

Comment (aljones15) [reply test/latest/10-canonicalize.js]: I believe this was addressed in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/b7a8b329f27a3681eaa4c579444fe1cd85ac45b6

File: implementations/implementation.hbs (hunk 3 above)
Comment (aljones15) [reply implementations/implementation.hbs]: Addressed in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/f791a4746422086bc8a91bda4d1292eb935eea51

File: implementations/generate.js (hunk 2 above)
Comment (aljones15) [reply implementations/generate.js]: I don't mind the extra white space especially in a giant scroll of test results, but addressed this in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/f791a4746422086bc8a91bda4d1292eb935eea51

**Pull request merged**